reviewing/plugin-commands-outdated/test/index.ts

```diff
@@ -409,6 +410,25 @@ test('pnpm outdated: catalog protocol', async () => {
 `)
 })
 
+test('pnpm outdated: --compatible works with npm aliases', async () => {
```

Comment:
Before the fix in `reviewing/outdated/src/outdated.ts`, the `--compatible` flag wouldn't work as expected when using npm aliases. This test failed before the fix.

<img width="842" alt="Screenshot 2024-12-01 at 2 58 34 PM" src="https://github.com/user-attachments/assets/c8f39aaf-2228-4655-b99a-f97056731e9a">